Whiskey/Tasks/Set-WhiskeyProGetAsset.ps1

```diff
@@ -0,0 +1,97 @@
+
+
+function Set-WhiskeyProGetAsset {
+    <#
+        .SYNOPSIS
+        Adds and Updates assets to the ProGet asset manager. 
+
+        .DESCRIPTION
+        The `Set-WhiskeyProGetAsset` adds assets to ProGet. A root directory needs to be 

```

Comment:
Use the task's name instead of the function name, e.g. ``The `Set-WhiskeyProGetAsset` `` should be ``The `PublishProGetAsset` task``.

Reply:
Done, I also changed the function name to be `Publish-WhiskeyProGetAsset` to keep it consistent.